### PR TITLE
Update StreamingVideo.md to fix the video scaling.

### PR DESCRIPTION
**Pull request by matinlotfali**

### CODE REVIEW

File: content/StreamingVideo.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ H="${3:-1080}"
 ./v4l2-init.sh
 echo "Streaming to ${1} with resolution ${W}x${H}..."
 
-gst-launch-1.0 v4l2src device=/dev/video0 \
+gst-launch-1.0 v4l2src device=/dev/video0 ! video/x-raw, width=1920, height=1080 \
```
Comment (mistyron) [inline content/StreamingVideo.md:81]: How did you choose the resolution 1920x1080?

**Review by mistyron — COMMENTED** [review]
requesting feedback about the chosen resolution

**Review by mistyron — CHANGES_REQUESTED** [review]
Why merge into master and not develop? It's not a hotfix, is it? Please refer to: 
image

File: content/StreamingVideo.md (hunk 1 above)
Comment (matinlotfali) [reply content/StreamingVideo.md:81]: It is the default resolution that is used in `./v4l2-init.sh`

**Comment by matinlotfali** [comment]
> Why merge into master and not develop?

I am sorry. Fixed it.

File: content/StreamingVideo.md (hunk 1 above)
Comment (mistyron) [reply content/StreamingVideo.md:81]: ok, that works. Thanks

**Review by mistyron — APPROVED** [review]
LGTM

**Pull request merged**